src/Microsoft.PowerShell.Security/security/AclCommands.cs

```diff
@@ -614,6 +614,7 @@ public static string GetSddl(PSObject instance)
         private string[] _exclude = new string[0];
     }
 
+#if !UNIX
```

Comment:
I'd like to see comments added explaining why it is excluded. It'll help us determine when it is appropriate to remove these ifdefs. This comment applies to all the new "#if !UNIX" additions.